.buildkite/test-pipeline.yaml

```diff
@@ -207,6 +207,23 @@ steps:
   command: pytest -v -s lora --shard-id=$$BUILDKITE_PARALLEL_JOB --num-shards=$$BUILDKITE_PARALLEL_JOB_COUNT --ignore=lora/test_long_context.py
   parallelism: 4
 
+- label: "PyTorch Fullgraph Smoke Test"
+  mirror_hardwares: [amd]
+  fast_check: true
+  source_file_dependencies:
+  - vllm/
+  - tests/compile
+  commands:
+  - pytest -v -s compile/test_full_graph_smoke.py
+
+- label: "PyTorch Fullgraph Test"
+  mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  - tests/compile
+  commands:
+  - pytest -v -s compile/test_full_graph.py
```

Comment:
we can merge them together into one suit.

Reply:
I wanted to keep these separate because the smoke test runs with every PR and only takes a couple minutes to run.  The other full test takes significantly longer.

Reply:
makes sense.